proofs.md

```diff
@@ -141,15 +155,17 @@ VerifyPoSt
   commRs        [POST_SECTORS_COUNT][32]byte,        // the commRs provided to GeneratePoSt
   challengeSeed [32]byte,
   faults        []uint64
-  proof         []byte,            // Multi-SNARK proof returned by GeneratePoSt
- ) err Error |
-  isValid bool                     // true iff the provided Proof of Space-Time is valid
+  proof         []byte,            // Multi-SNARK proof returned by GeneratePoSt 
+ ) err Error | 
+  isValid bool                     // true iff the provided Proof of Spacetime is valid
 ```
 
----
+------
+
 ## Piece Inclusion Proof
 
 ### PieceInclusionProof
```

Comment:
Feels like we could just inline that text since this subsection has the same name as its parent section "Piece Inclusion Proof"